en/shapes.md

```diff
@@ -1,4 +1,5 @@
 ### shapes.txt
+Should be provided in every case.
```

Comment:
Elaborate on what a "case is" – see proposal in https://github.com/MobilityData/GTFS_Schedule_Best-Practices/pull/38